### ISSUE TITLE
Can't call os.path.realpath.

### ISSUE THREAD

**Issue by kayabaNerve** [microsoft/pyright#298]
**Describe the bug**
The os.path.realpath always complains about being passed the wrong type, even if the passed argument is a `str`, `Path`, or `PurePath` (despite all being valid options).

**Expected behavior**
Pyright should acknowledge any of the above as valid options to be passed.

**Screenshots or Code**
```
from os import path
from pathlib import PurePath, Path
file: str = __file__
realPath: str = path.realpath(file)
realPath: str = path.realpath(Path(file))
realPath: str = path.realpath(PurePath(file))
```

**Additional context**
```
  4:31 - error: Argument of type 'str' cannot be assigned to parameter 'path' of type '_PathLike[AnyStr]'
  'str' is incompatible with '_PathLike[AnyStr]'
  4:31 - error: Argument of type 'str' cannot be assigned to parameter 'filename' of type '_PathLike[AnyStr]'
  'str' is incompatible with '_PathLike[AnyStr]'
  5:31 - error: Argument of type 'Path' cannot be assigned to parameter 'path' of type 'TypeVar['AnyStr', str, bytes]'
  Type 'Path' is not compatible with constraints imposed by TypeVar 'AnyStr'
  5:31 - error: Argument of type 'Path' cannot be assigned to parameter 'filename' of type 'TypeVar['AnyStr', str, bytes]'
  Type 'Path' is not compatible with constraints imposed by TypeVar 'AnyStr'
  6:31 - error: Argument of type 'PurePath' cannot be assigned to parameter 'path' of type 'TypeVar['AnyStr', str, bytes]'
  Type 'PurePath' is not compatible with constraints imposed by TypeVar 'AnyStr'
  6:31 - error: Argument of type 'PurePath' cannot be assigned to parameter 'filename' of type 'TypeVar['AnyStr', str, bytes]'
  Type 'PurePath' is not compatible with constraints imposed by TypeVar 'AnyStr'
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now fixed in 1.0.80, which I just published.
